backend/src/bundles/auth/auth.controller.ts

```diff
@@ -62,8 +97,10 @@ class AuthController extends BaseController {
 
     /**
      * @swagger
-     * /auth/sign-up:
+     * /api/v1/auth/sign-up:
```

Comment:
The /api/v1 part of the URL is redundant, other versions will be available at different URLs, such as /v2/documentation etc

Reply:
or maybe not) 
i'm not sure if we will be able to send requests in swagger without this part. double-check pls

Reply:
Unfortunately, we won't be able to send request without this part: `/api/v1/`